src/internal/m365/onedrive/restore.go

```diff
@@ -851,7 +851,7 @@ func restoreData(
 		}
 
 		collision = dci
-		replace = restoreCfg.OnCollision == control.Replace && isFolder == dci.IsFolder
+		shouldDeleteOriginal = restoreCfg.OnCollision == control.Replace && isFolder == dci.IsFolder
```

Comment:
this may disappear when rebasing, but this doesn't need the `isFolder` check because it'll always be false

Reply:
Ahah, fixed that in exchange and forgot onedrive.  Thanks!